larg4/pluginActions/ParticleListAction_service.cxx

```diff
@@ -563,21 +569,29 @@ namespace larg4 {
     } // end if this is the first step
 
     // At this point, the particle is being transported through the
-    // simulation. This method is being called for every voxel that
+    // simulation.
+    // change below:
+    // This method is being called for every step that
     // the track passes through, but we don't want to update the
     // trajectory information if we're just updating voxels. To check
     // for this, look at the process name for the step, and compare it
     // against the voxelization process name (set in PhysicsList.cxx).
     G4String process = step->GetPostStepPoint()->GetProcessDefinedStep()->GetProcessName();
-    G4bool ignoreProcess = process.contains("LArVoxel") || process.contains("OpDetReadout");
-
+    G4bool ignoreProcess = process.contains("StepLimiter");
+    
+    //  if (ignoreProcess)
+    //  {
+    //	std::cout <<"StepLimit ********************* "<<std::endl;
+    //  }
+   
+    // mf::LogDebug("ParticleListActionService::SteppingAction")
     /*
-    mf::LogDebug("ParticleListActionService::SteppingAction")
-    << ": DEBUG - process='"
+    std::cout
+    << ": DEBUG - process="
```

Comment:
I would restore the `mf::LogDebug` statement here.

Reply:
done